CreatubblesAPIClient/Sources/Model/ToybooCreation.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  ToybooCreation.swift
+//  CreatubblesAPIClient
+//
+//  Created by Nomtek on 02.11.2016.
+//  Copyright © 2016 Nomtek. All rights reserved.
+//
+
+import UIKit
+
+@objc
+public class ToybooCreation: NSObject
+{
+    public let identifier: String
```

Comment:
are you sure, that `uzpbUrl` and `contentUrl` are not optional? (They're not documented anywhere).